Lib/tempfile.py

```diff
@@ -643,6 +643,17 @@ def __init__(self, max_size=0, mode='w+b', buffering=-1,
                                    'encoding': encoding, 'newline': newline,
                                    'dir': dir, 'errors': errors}
 
+    def __class_getitem__(cls, type):
+        """Provide minimal support for using this class as generic
+        (for example in type annotations).
+
+        See PEP 484 and PEP 560 for more details. For example,
+        `SpooledTemporaryFile[str]` is a valid expression at runtime (type
+        argument `str` indicates the type used for mode). Note, no type
```

Comment:
This is not clear, say something like "the type argument indicates whether the file is open in bytes or text mode".